node_modules/oae-core/upload/js/upload.js

```diff
@@ -71,7 +82,7 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport', 'j
             $('#upload-modal > .modal-footer', $rootel).show();
 
             // Reset controls
-            $('#upload-modal *', $rootel).prop('disabled', false);
+            $toDisable().prop('disabled', false);
```

Comment:
As this is enabling the controls, there's no need to exclude the file input element.